snuba/migrations/snuba_migrations/stacktraces/0001_stacktraces.py

```diff
@@ -0,0 +1,95 @@
+from typing import List, Sequence
+
+from snuba.clickhouse.columns import (
+    UUID,
+    Column,
+    DateTime,
+    String,
+    UInt,
+    Tuple,
+    NamedTuple,
+)
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations, table_engines
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+
+columns: List[Column[Modifiers]] = [
+    # primary key
+    Column("project_id", UInt(64)),
+    Column("transaction_id", UUID()),
+
+    # profiling data
+    Column("stacktrace", String(Modifiers(codecs=["LZ4HC(9)"]))),
+    Column("symbols", String(Modifiers(codecs=["LZ4HC(9)"]))),
+
+    # filtering data
+    Column("trace_id", UUID()),
+    Column("transaction_name", String(Modifiers(low_cardinality=True))),
+    Column("environment", String(Modifiers(nullable=True, low_cardinality=True))),
+
+    Column("version", NamedTuple((("name", String()), ("code", String())), Modifiers(low_cardinality=True))),
+    Column("platform", String(Modifiers(low_cardinality=True))),
+    Column("android_api_level", UInt(32, Modifiers(nullable=True))),
+    Column("device_classification", String(Modifiers(low_cardinality=True))),
+    Column("device_locale", String(Modifiers(low_cardinality=True))),
+    Column("device_manufacturer", String(Modifiers(low_cardinality=True))),
+    Column("device_model", String(Modifiers(low_cardinality=True))),
+    Column("device_os_build_number", String(Modifiers(low_cardinality=True))),
+    Column("device_os_name", String(Modifiers(low_cardinality=True))),
+    Column("device_os_version", String(Modifiers(low_cardinality=True))),
+
+    Column("error_code", String(Modifiers(low_cardinality=True, nullable=True))),
+    Column("error_description", String(Modifiers(low_cardinality=True, nullable=True))),
+
+    Column("duration_ns", UInt(64)),
```

Comment:
Does this mean nanoseconds? Is that useful? I believe duration in other tables like transactions is in milliseconds.

Reply:
It does mean nanoseconds and it's a carry-over from the Specto SDKs where we would send nanoseconds. I'll check with my team but we can likely change that to milliseconds if you insist.

Reply:
Definitely not insisting, it doesn't really make a ton of difference here.. apart from maybe being a UInt32 vs UInt64. Asking the question more from the product perspective really - do we want to present different granularities for each type of product in the UI or would we want to keep them uniform?

Reply:
We'd be converting to milliseconds for sure before displaying that, I think we kept that to work with integers only until display. I think we'd like to keep that and we need to store it as a `UInt64` otherwise it would represent only max of 4s.

Reply:
👍